PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -534,18 +535,29 @@ BOOL WindowsDlg::onInitDialog()
 	lvColumn.mask = LVCF_WIDTH | LVCF_TEXT | LVCF_SUBITEM | LVCF_FMT;
 	lvColumn.fmt = LVCFMT_LEFT;
 
-	lvColumn.pszText = TEXT("\u21F5 Name");
+	CString columnText;

```

Comment:
generic_sting would be the normal choice for n++
ATL is not available for mingw.
See http://en.cppreference.com/w/cpp/string/basic_string/data regarding returning non const string data pointer. 
What about a const_cast?

Reply:
I am not a fan of `const_cast` personally. Do we have non-const `data()` available in VS2015?